soccer/gameplay/tests/test_circle_near_ball.py

```diff
@@ -0,0 +1,76 @@
+import unittest
+import main
+import robocup
+import math
+import tactics.stopped.circle_near_ball
+import constants
+
+class Moc_Ball:
+	def __init__(self, x, y):
+		self.pos = robocup.Point(x, y)
+
+class TestCircleNearBall(unittest.TestCase):
+	def __init__(self, *args, **kwargs):
+		super(TestCircleNearBall, self).__init__(*args, **kwargs)
+
+	def setUp(self):
+		self.circle_near_ball = tactics.stopped.circle_near_ball.CircleNearBall()
+
+	def test_get_circle_points(self):
+		width = constants.Field.Width
+		length = constants.Field.Length
+		
+		# Checks that the points around the ball are within the bounds of the field
+		def assert_circle_points(self):
+			array = self.circle_near_ball.get_circle_points(6)
+			for point in array:		
+				self.assertTrue(point.x >= width * -1 / 2 and point.x <= width / 2, msg="failure of get_circle_points. X position of points are outside the field. Point's x position is %d" %point.x)
+				self.assertTrue(point.y >= 0 and point.y <= length, msg="failure of get_circle_points. Y positions of points are outside the field. Point's y position is %d" %point.y )
```

Comment:
These should be split up into `assertLess` and `assertGreater`. When trying to debug this and it fails later on, it will tell us what the actual values of the variables instead of `Error: Expected True, Received False`